### PR TITLE
Fix set lagos as default

### LINKED ISSUE #22: Set default choice in state for deal model to be Lagos
The default choice of state for deal model should be Lagos.

### PR DESCRIPTION
**Pull request by andela-osule**
Fixes #22

### CODE REVIEW

File: troupon/deals/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+from deals.models import Deal, Advertiser, Category
+
+
+class DealModelTestCase(TestCase):
+
+    def setUp(self):
+        advertiser, category = Advertiser(1), Category(1)
```
Comment (andela-cnnadi) [inline troupon/deals/tests/test_models.py]: Can this be changed?

**Comment by andela-osule** [comment]
Yes. Should I close and reopen this?

**Comment by andela-cnnadi** [comment]
Is this still required?

**Comment by andela-osule** [comment]
Yes, it fixes lagos as the default state in model.

**Pull request closed without merging**